subprojects/gradle/build-verdict/src/main/kotlin/com/avito/android/build_verdict/TaskErrorOutputCaptureExecutionListener.kt

```diff
@@ -44,6 +44,15 @@ internal class TaskErrorOutputCaptureExecutionListener(
     override fun afterExecute(task: Task, state: TaskState) {
         if (state.failure == null) {
             logs.remove(Path.path(task.path))
+        } else {
+            when (task) {
+                is BuildVerdictTask -> {
```

Comment:
Do we really want to log all these tasks?
Why do we do it if we have a contract `BuildVerdictTask`?